buildenv/docker/jdk8/ppc64le/ubuntu16/Dockerfile

```diff
@@ -0,0 +1,88 @@
+# Copyright (c) 2017, 2017 IBM Corp. and others
+#
+# This program and the accompanying materials are made available under
+# the terms of the Eclipse Public License 2.0 which accompanies this
+# distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+# or the Apache License, Version 2.0 which accompanies this distribution and
+# is available at https://www.apache.org/licenses/LICENSE-2.0.
+#
+# This Source Code may also be made available under the following
+# Secondary Licenses when the conditions for such availability set
+# forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+# General Public License, version 2 with the GNU Classpath
+# Exception [1] and GNU General Public License, version 2 with the
+# OpenJDK Assembly Exception [2].
+#
+# [1] https://www.gnu.org/software/classpath/license.html
+# [2] http://openjdk.java.net/legal/assembly-exception.html
+#
+# SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+
+# To use this docker file:
+#   docker build -t=openj9 .
+#   docker run -it openj9
+
+FROM ubuntu:16.04
+
+# Get prerequisites for add-apt-repository
+RUN apt-get update \
+  && apt-get install -qq -y --no-install-recommends \
+    software-properties-common \
+    python-software-properties
+
+# Add apt-get repository containing openjdk7
+RUN apt-get update \
+  && add-apt-repository ppa:openjdk-r/ppa
+
+# Install required OS tools
+RUN apt-get update \
```

Comment:
There are 3 run commands here and each one creates a new layer in the Docker image.  Only the 3rd one deletes the var lists (`rm -rf /var/lib/apt/lists/*`) so the final image will be larger than necessary.

Can you merge the 3 apt-get sections into one?

Reply:
Please also sort the dependencies alphabetically when merging into one apt-get section

Reply:
-- Can you merge the 3 apt-get sections into one?

Unfortunately no. We need the first apt-get to get the tools required for the apt-add, and the third apt-get includes openjdk7, which will only be found after we've added the new repository and updated.

Reply:
I may have misunderstood you. By "one section", did you mean several commands with lots of &'s?

e.g:
update 
&& apt get
&& apt add
&& update
&& apt get

Reply:
Yes - one command with many &&'s as this creates 1 layer in the docker image vs 3 which means it's a smaller download.